src/ResourceManager/Reservations/Commands.Reservations/Cmdlets/GetReservation.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.Reservations.Common;
+using Microsoft.Azure.Management.Reservations.Models;
+using Microsoft.Azure.Commands.Reservations.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Management.Reservations;
+
+namespace Microsoft.Azure.Commands.Reservations.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Get, "AzureRmReservation", DefaultParameterSetName = Constants.ParameterSetNames.CommandParameterSet), OutputType(typeof(PSReservationPage), typeof(PSReservation))]
```

Comment:
You should use a single OutputType attribute if all parameter sets return the same type

Reply:
this command actually returns two different types, depending whether ReservationId was provided (single reservation is returned) or not (multiple reservations are returned)